src/strategies/mod.rs

```diff
@@ -3,24 +3,22 @@
 //! Rank 1 Constraint Systems (R1CS).
 //!
 //! The inputs of the permutation function have to be explicitly
-//! over the Scalar Field of the curve25519 so working over
+//! over the DalekScalar Field of the curve25519 so working over
 //! `Fp = 2^252 + 27742317777372353535851937790883648493`.
```

Comment:
That's not correct, We are working over `BLS12_381` Scalar field.

Reply:
Yep, I didn't update the documentation yet. I'll check onto that